web_src/js/features/common-global.js

```diff
@@ -302,51 +302,55 @@ async function linkAction(e) {
 }
 
 export function initGlobalLinkActions() {
-  function showDeletePopup(e) {
-    e.preventDefault();
-    const $this = $(this);
-    const dataArray = $this.data();
-    let filter = '';
-    if (this.getAttribute('data-modal-id')) {
-      filter += `#${this.getAttribute('data-modal-id')}`;
-    }
+  for (const el of document.querySelectorAll('.delete-button')) {
+    el.addEventListener('click', (e) => {
+      e.preventDefault();
+      const btn = e.currentTarget;
```

Comment:
It could be wrong if there is a SVG in the button.

Reply:
No, as I understand, `e.currentTarget` is always equal to `this`, e.g. the element where the listener is bound:

https://stackoverflow.com/a/5921528/808699

Reply:
Sorry, misread.